yt_dlp/extractor/samplefocus.py

```diff
@@ -82,7 +82,15 @@ def extract_count(klass):
         return {
             'id': sample_id,
             'title': title,
-            'url': mp3_url,
+            'formats': [{
+                'url': mp3_url,
+                'ext': 'mp3',
+                'vcodec': 'none',
+                'acodec': 'mp3',
+                'http_headers': {
+                    'Referer': url,
+                },
+            }],
```

Comment:
these could just be in the info dict root instead of a formats dict, but either way works. core code will also determine acodec from ext, but explicitly returning it is fine too